app/controllers/submissions_controller.rb

```diff
@@ -13,8 +13,10 @@ def create
     end
   end
 
+  # Clear session unless this was an ECP policy. If ECP policy, user has
+  # oppotunity to set a reminder up where name and email is re-used from claim.
```

Comment:
@d-a-v-e Should we be clearing down the `current_claim_session` for ECP at some point later in the journey, so that it can be used, but is actually cleared eventually?

Reply:
not sure i understand the Q, but the session is cleared later on in the reminders flow: https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/1558/files#diff-b8e469ddb63fc4efe9e712d5b2f68490f9e3b4bc9d055bc312dd42e90ddb8402R21